triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/simulation/TriggerMultiProcessSimulation.scala

```diff
@@ -54,26 +54,27 @@ abstract class TriggerMultiProcessSimulation extends AsyncWordSpec with Abstract
 
       Behaviors
         .supervise[Message] {
-          Behaviors.setup { context =>
+          Behaviors.setup { rootContext =>
```

Comment:
Changes here simplify understanding what Actor context is being used (and so what child actors are watched by the parent)